lib/airbrake.js

```diff
@@ -368,6 +261,50 @@ Airbrake.prototype.cgiDataVars = function(err) {
   return cgiData;
 };
 
+Airbrake.prototype.contextJSON = function(err){
+  var context = {};
+  context["notifier"] = {
+    name: Airbrake.PACKAGE.name,
+    version: Airbrake.PACKAGE.version,
+    url: Airbrake.PACKAGE.homepage
+  };
+  context["environment"] = this.env;
+  context["rootDirectory"] = this.projectRoot;
+  context["os"] = os.type();
+  context["hostname"] = os.hostname();
+  if (err.url && err.url.substr(0, 1) === '/') {
+    context["url"] = this.host + err.url;
+  } else if (err.url){
+    context["url"] = err.url;
+  } else {
+    context["url"] = this.host;
+  }
+  return context;
+};
+
+Airbrake.prototype.notifyJSON = function(err){
+  var trace = stackTrace.parse(err);
+  var self = this;
+  return stringify({
+    "errors": [
+      {
+        type: err.type || "Error",
+        message: err.message || "-",
```

Comment:
I think it's fine to send empty error message. The dashboard can handle this (you'll see `<empty error message>`.